hokusai/production.yml

```diff
@@ -21,13 +20,33 @@ spec:
     spec:
       containers:
         - name: aprb-web
+          env:
+            - name: PORT
+              value: "4000"
           envFrom:
-          - configMapRef:
-              name: aprb-environment
+            - configMapRef:
+                name: aprb-environment
           image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/aprb:production
           imagePullPolicy: Always
           ports:
-          - containerPort: 4000
+            - name: aprb-http
+              containerPort: 4000
+          readinessProbe:
+            httpGet:
+              port: aprb-http
+              path: /ping
+              httpHeaders:
+                - name: X-FORWARDED-PROTO
+                  value: https
+            initialDelaySeconds: 5
+            periodSeconds: 15
+            timeoutSeconds: 10
+          resources:
```

Comment:
do we have to set these limits? my guess is this app uses very limited CPU and memory, maybe we only need to set some low limits?

Reply:
These should be set on everything, but potentially these could be lower -- will check out some other smaller apps to see how low they get, maybe @izakp has ideas on a better minimum baseline.

Reply:
Bumped these down significantly